src/app/app-modules/login/login.component.ts

```
@@ -20,7 +20,7 @@
  * along with this program.  If not, see https://www.gnu.org/licenses/.
  */
 
-import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
+import { Component, OnInit, AfterViewInit, ViewChild, ElementRef } from '@angular/core';
```

ESLint

[error] Replace `·Component,·OnInit,·AfterViewInit,·ViewChild,·ElementRef·` with `⏎··Component,⏎··OnInit,⏎··AfterViewInit,⏎··ViewChild,⏎··ElementRef,⏎`

(prettier/prettier)